Sources/Afterpay/Model/ShippingOption.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ShippingOption.swift
+//  Afterpay
+//
+//  Created by Adam Campbell on 7/12/20.
+//  Copyright © 2020 Afterpay. All rights reserved.
+//
+
+import Foundation
+
+public struct ShippingOption: Codable {
```

Comment:
Same with these structs.

(Not that it's my preference—just for consistency 😛)